### DISCUSSION TITLE
linting Problems with cSpell

### DISCUSSION THREAD

**Discussion by DennisWarchocki** [neuefische/questions#179] [Web] [answered]
Hello guys,

In my Capstone Project, I get plenty of Problems displayed all caused by cSpell. He's telling me 
"stylelint": Unknown word (cSpell)

So right now I get 20 Problems displayed all about that cSpell linting.
Its a bit distracting if I want to track real Problems within my App.

My question would be, why cSpell was implented in the Capstone template? And how I could change that behaviour or how to delete cSpell from my repo?

<img width="545" alt="Bildschirmfoto 2022-09-28 um 15 13 27" src="https://user-images.githubusercontent.com/109968361/192789324-0f958573-177a-4379-90f1-979ebaa13c6d.png">

Link to my repo: 
https://github.com/DennisWarchocki/capstone-project

**Comment by actyralf** [comment, marked as answer]
I think this isn't a linter problem, it seems that the VSCode extension code-spell-checker is responsible for these warnings.
You can disable or uninstall it via VSCode extensions.

**Reply by DennisWarchocki** [reply]
That was the solution.
merci (:
